test/integration/compAnchorIntegration.js

```diff
@@ -0,0 +1,559 @@
+    /*
+    *  user2 deposit on dest chain
+    */
+    // generate deposit commitment
+    destDeposit = helpers.generateDeposit(originChainID);
+    // deposit on dest chain and define nonce
+    ({logs} = await DestChainLinkableAnchorInstance.deposit(helpers.toFixedHex(destDeposit.commitment), {from: user2}));
+    destUpdateNonce = logs[0].args.leafIndex;
+    destMerkleRoot = await DestChainLinkableAnchorInstance.getLastRoot();
+    // create correct update proposal data for the deposit on dest chain
+    destDepositData = helpers.createUpdateProposalData(destChainID, destBlockHeight, destMerkleRoot);
+    destDepositDataHash = Ethers.utils.keccak256(OriginAnchorHandlerInstance.address + destDepositData.substr(2));
+    /*
+    *  relayers vote on origin chain
+    */
+    // deposit on dest chain leads to update proposal on origin chain
+    // relayer1 creates the deposit proposal
+    await TruffleAssert.passes(OriginBridgeInstance.voteProposal(
+      destChainID,
+      destUpdateNonce,
+      resourceID,
+      destDepositDataHash,
+      { from: relayer1Address }
+    ));
+    // relayer1 will execute the update proposal
+    await TruffleAssert.passes(OriginBridgeInstance.executeProposal(
+      destChainID,
+      destUpdateNonce,
+      destDepositData,
+      resourceID,
+      { from: relayer1Address }
+    ));
+    // check initial balances
+    balanceOperatorBefore = await originWrapperToken.balanceOf(operator);
+    balanceReceiverBefore = await originWrapperToken.balanceOf(user2);
+    // get roots for proof
+    const originNeighborRoots = await OriginChainLinkableAnchorInstance.getLatestNeighborRoots();
+    /*
+    *  user2 generates proof
+    */
+    tree = new MerkleTree(merkleTreeHeight, null, prefix)  
+    await tree.insert(destDeposit.commitment);
+    
+    ({ root, path_elements, path_index } = await tree.path(0));
+    const originNativeRoot = await OriginChainLinkableAnchorInstance.getLastRoot();
+    input = {
+      // public
+      nullifierHash: destDeposit.nullifierHash,
+      recipient: user2,
+      relayer: operator,
+      fee,
+      refund,
+      chainID: destDeposit.chainID,
+      roots: [originNativeRoot, ...originNeighborRoots],
+      // private
+      nullifier: destDeposit.nullifier,
+      secret: destDeposit.secret,
+      pathElements: path_elements,
+      pathIndices: path_index,
+      diffs: [originNativeRoot, originNeighborRoots[0]].map(r => {
+        return F.sub(
+          Scalar.fromString(`${r}`),
+          Scalar.fromString(`${originNeighborRoots[0]}`),
+        ).toString();
+      }),
+    };
+
+    wtns = await createWitness(input);
+
+    res = await snarkjs.groth16.prove('test/fixtures/circuit_final.zkey', wtns);
+    proof = res.proof;
+    publicSignals = res.publicSignals;
+    vKey = await snarkjs.zKey.exportVerificationKey('test/fixtures/circuit_final.zkey');
+    res = await snarkjs.groth16.verify(vKey, publicSignals, proof);
+    assert.strictEqual(res, true);
```

Comment:
these can be removed from integration tests.

Reply:
Great. Thank you